lp-builder-config/misc.yaml

```diff
@@ -153,7 +167,7 @@ projects:
         build-channels:
           charmcraft: "1.5/stable"
         channels:
-          - latest/edge
+          #- latest/edge
```

Comment:
shouldn't master still be latest/edge and the channel `jammy/edge` be built from `stable/jammy`?

Reply:
TBH, I'm not sure.  I feel really conflicted about where a charm should push code for jammy charms.  One view I have is that we'll be in the 'jammy' cycle for the next two years.  The other view is that we were in the jammy dev cycle for the last 6 months, and now it is released, it is stable, and the charm's master branch is actually going to target the next release (k=j+1).  And that works against that, but that jammy is stable.  I then you are tending towards the latter view, and I'm happy to endorse that too.  Is that what you were thinking?

Reply:
yes, in my view of the world the 22.04 marks the point where `jammy` gets stable, although the question "what will happen with the k devel cycle?", ... do we want to have a 1-to-1 relationship between the git branch name and the track name?, that's probably the source of the doubts, I personally think we shouldn't, track names are meant to be user consumable labels while git branches names are development milestones.

thinking out loud here: *maybe* this is a charm where we could introduce the use of tags, so now `stable/jammy` git branch gets pushed into the `jammy/edge` charmhub track, then in 6 months from now we just cut a new release making a tag (22.10-b1 for betas, 22.10-rc1 for candidate, 22.10.0 for stable) ... this kinda changes the way we designed things until now though.

Reply:
Okay, I've updated the PR to have a `stable/jammy` branch in github feed the 'yammy/edge' channel in charmhub.  Thanks for helping resolve this.